### PR TITLE
Translations

### PR DESCRIPTION
**Pull request by KjellMorgenstern**
Updated the translations with qt lupdate,
resolved some conflicts, reuse some translation with minor changes in the English version, restructed the translator credits

### CODE REVIEW

File: src/help/aboutbox.cpp [hunk 1]
```diff
@@ -139,23 +139,24 @@ QString("<br /><br /><br /><br /><br /><br /><br /><br /><br />") +
...
+	tr("Japanese: ") + tr("Hiroshi Suzuki ") +
+	tr("Russian: ") + "Vladimir Savinov " +
+	tr("Dutch: ") + "Steven Noppe, Davy Uittenbogerd " +
+	tr("Czech: ") + "Josef Dustira " +
+	tr("Korean: ") + tr("Jinbuhm Kim ") +
+	tr("Slovak: ") + " &#313;ubom&iacute;r Ducho&#328; " +
+    tr("Greek: ") + "Alexander Kaltsas " +
+    tr("French: ") + "Lionel Michel, Yvan Kloster, Alexandre Dussart, Roald Baudoux " +
+    tr("Turkish: ") + "Cihan Mete Bahad&#x0131;r " +
+    tr("Bulgarian: ") + "Nikolay Stankov, Lyubomir Vasilev " +
```
Comment (ovidiub13) [inline src/help/aboutbox.cpp]: Keep the same indentation for all lines please. Also, sorting the languages alphabetically would be nice.

Comment (ovidiub13) [reply src/help/aboutbox.cpp]: maybe also some vertical alignment, on columns, languages and names, it would be even clearer.

Comment (ovidiub13) [reply src/help/aboutbox.cpp]: Not all names are put in `tr()`, this breaks the translation files, most languages are missing the names.

Names shouldn't be translated IMO. So we should avoid using `and` or `&` in names strings, and just use commas.

We should not put names in `tr()`. Exception could be if traditionally names are rewritten in the non latin alphabets, Japanese, russian, greek, etc.

Comment (ovidiub13) [reply src/help/aboutbox.cpp]: And this is exactly what you did... Sorry, I just woke up. Perhaps do the same for Greek, Russian, Bulgarian, ?

**Comment by KjellMorgenstern** [comment]
I put Asian names into tr because they often give themselves western names, that can be easily remembered and pronounced by foreigners.
Regarding the Unicode, this is a point, too. 

None of this yet is used in the names, the Chinese translators did not add their Chinese names, or they already got lost from the files in the past, so i‘d also be fine with YAGNI. Maybe it’s ok to use Unicode in the .cpp file?  I think all tools should support it (it’s supported with C++11)

**Pull request merged**